src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
+    """
+    The reduction that sometimes is used to lodge the substrate in the substrate holder position..
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    insert_id = Quantity(
+        type=str,
+        description="""
+        The ID of the insert that is placed in this position to accomodate the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    inner_geometry = SubSection(
+        section_def=Geometry,
+    )
+    outer_geometry = SubSection(
+        section_def=Geometry,
+    )
+
+class SubstrateHolderPosition(ArchiveSection):
+    """
+    One casing position of the substrate holder.
+    """
+    name = Quantity(
```

Comment:
Is this name also used as the label? If we will use this to refer to the position later we should maybe mention that in the description.

Reply:
Yes. I will specify it.